src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmClrType.cs

```diff
@@ -44,7 +44,9 @@ internal DkmClrType(DkmClrModuleInstance module, DkmClrAppDomain appDomain, Type
             ModuleInstance = module;
             AppDomain = appDomain;
             _lmrType = lmrType;
-            _evalAttributes = GetEvalAttributes(lmrType);
+            _lazyEvalAttributes = new System.Lazy<ReadOnlyCollection<DkmClrEvalAttribute>>(
+                () => GetEvalAttributes(lmrType),
```

Comment:
`GetEvalAttribute` realizes a huge number of `string` and arrays inside the reflection libraries. About 150MB of savings from this change.